infrastructure/code/code_html.go

```diff
@@ -68,11 +108,91 @@ func getDetailsHtml(issue snyk.Issue) string {
 
 	dataFlowHtml := getDataFlowHtml(additionalData)
 
+	// Header
 	html := replaceVariableInHtml(detailsHtmlTemplate, "issueId", issue.ID)
 	html = replaceVariableInHtml(html, "issueTitle", additionalData.Title)
+	html = replaceVariableInHtml(html, "issueType", getIssueType(additionalData))
 	html = replaceVariableInHtml(html, "severityText", issue.Severity.String())
-	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+	html = replaceVariableInHtml(html, "severityIcon", getSeverityIconSvg(issue))
+
+	html = replaceVariableInHtml(html, "issueOverview", additionalData.Message)
+	// Data flow
 	html = replaceVariableInHtml(html, "dataFlowCount", fmt.Sprintf("%d", len(additionalData.DataFlow)))
+	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+
+	// External example fixes
+	html = replaceVariableInHtml(html, "repoCount", fmt.Sprintf("%d", additionalData.RepoDatasetSize))
+	html = replaceVariableInHtml(html, "exampleCount", fmt.Sprintf("%d", len(additionalData.ExampleCommitFixes)))
+	html = replaceVariableInHtml(html, "tabsNav", getTabsHtml(additionalData.ExampleCommitFixes))
 
 	return html
 }
+
+func getFileName(filePath string) string {
+	if filePath == "" {
+		return ""
+	}
+
+	// Windows paths
+	filePath = strings.Replace(filePath, "\\", "/", -1)
+
+	// Handle trailing slash
+	if strings.HasSuffix(filePath, "/") {
+		return ""
+	}
+
+	filename := filepath.Base(filePath)
+	return filename
+}
+
+func getIssueType(additionalData snyk.CodeIssueData) string {
+	if additionalData.IsSecurityType {
+		return "Vulnerability"
+	}
+	return "Quality Issue"
```

Comment:
I don't think we need to do this for `Quality Issue` types, but I could be wrong? The aim was to replace the Suggestion Panel for vulnerabilities that come up under `Code Security` in this screenshot. 
![Screenshot 2024-04-02 at 13 11 41](https://github.com/snyk/snyk-ls/assets/81559517/18ebd241-18e9-49b3-94a1-2461253f3f80)

Reply:
Hmm in that case leave this code as is. I think I need to understand this a bit better, I never even looked at the `Code Quality` section

Reply:
We should do both, I think :). We handle both as "Code"